services/src/main/java/org/fao/geonet/api/userfeedback/UserFeedbackUtils.java

```diff
@@ -45,144 +55,46 @@
      */
     public class RatingAverage {
 
-        /** The metadata title. */
-        private String metadataTitle;
```

Comment:
Without the title on the home page there is no reference to the record

![userfeedbackhome](https://user-images.githubusercontent.com/1323093/39309042-5e1bd430-4967-11e8-9015-c3bfb8c9a6b5.png)

Reply:
True so I will restore that but do not rely on 19139 xpath

Reply:
Works fine now